deploy-zirakerp.sh

```
@@ -0,0 +1,325 @@
+
+echo "=============================================="
+echo "  ZirakERP Deployment Starting..."
+echo "=============================================="
+
+# ========================
+# STEP 1: Stop & Remove n8n
+# ========================
+echo ""
+echo "[1/9] Stopping and removing n8n..."
+if command -v docker &>/dev/null; then
+    docker stop $(docker ps -aq) 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)